bluesky/examples.py

```diff
@@ -364,9 +398,10 @@ def kickoff(self):
         return self
 
     def describe(self):
-        return [{k: {'source': self.name, 'dtype': 'number'}
-                 for k in [self._motor, self._det]},
-                {self._det2: {'source': self.name, 'dtype': 'number'}}]
+        dd = dict()
+        for k in [self._motor, self._det, self._det2]:
+            dd.update(k.describe())
+        return dd
```

Comment:
While the return value from `reader.describe()` or `motor.describe()` is expected to be a dict, notice in the deleted lines here that `flyer.describe()` returns a *list* of dicts. That's because flyers can return data for multiple event streams (multiple descriptors). Each dict in the list corresponds to one set of data keys (one descriptor).

This has also caught @klauer, @tacaswell, and myself by surprise at various times. More evidence that `describe` on flyers probably needs a distinct name. But that's a separate issue. Fixing this to return a list should un-break Travis.